### ISSUE TITLE
Small correction for screenshot in Articles/MISPTISetup.md

### ISSUE THREAD

**Issue by hilton-ashford** [Cyberlorians/Articles#5]
Firstly, thank you for the hard work on the write-up.

In Step 2 of "Setup the script for MISP-Sentinel API Calls", the screenshot shows the value "alerts" for action.
This produces an error in the logs when running the batch import of the indicators.

"message": "Error converting value \"alerts\" to type 'System.Nullable`1[IsgWebApi.SecurityGraphModel.TiAction]"

This is caused because only the values "_alert_", "_allow_" and "_block_" are valid.
source: https://github.com/microsoftgraph/security-api-solutions/tree/master/Samples/MISP#action

I thought I would add this comment to help someone that might be following your guide to the letter T, and is not able to import the IOCs.

Keep up the great work!

**Comment by fpsilva-source** [comment]
Hi Hilton,

Did your installation succeed following the steps in this document?

**Comment by Cyberlorians** [comment]
Yup. Correct. Api changed and I forgot to change this week. I need to update. I'm very sorry.

**Comment by Cyberlorians** [comment]
I have a request into our product group too. The app reg says to use Microsoft sentinel but azure sentinel only does. Little misleading

**Comment by Cyberlorians** [comment]
Changes have been updated.

**Comment by hilton-ashford** [comment]
> Hi Hilton,
> 
> Did your installation succeed following the steps in this document?

Hi fpsilva-source, the install went through perfectly for the rest, it was just that part of the guide that didn't work, but I see the .md was updated to correct it.

The only other issue I experienced was due to the server being installed on Hyper-V, and the listening port for xrdp was set to listen on vsock and not TCP (in the xrdp.ini), but this is platform specific and not to do with the procedure.

Thank you Cyberlorians for following up, and for the hard work, it helped us set up a working solution.
